### PR TITLE
#4 Added Tab Five

### LINKED ISSUE #4: Make Tab Five
Create a fifth template tab

### PR DESCRIPTION
**Pull request by tgeller08**
## Description

I added a fifth template tab.

Fixes  #4 

## Screenshots

IMG_4559

## Checklist

- [x] tested locally
- [x] added reviewers

### CODE REVIEW

**Comment by rosiesr** [comment]
LGTM!!! :)

File: screens/TabFiveScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import { StyleSheet } from 'react-native';
+
+import EditScreenInfo from '../components/EditScreenInfo';
+import { Text, View } from '../components/Themed';
+
+export default function TabThreeScreen() {
```
Comment (annazhang2002) [inline screens/TabFiveScreen.tsx:6]: Don't forget to change this to be `TabFiveScreen`!

**Review by annazhang2002 — CHANGES_REQUESTED** [review]
just one quick change but looks good to go!

File: screens/TabFiveScreen.tsx (hunk 1 above)
Comment (tgeller08) [reply screens/TabFiveScreen.tsx]: Ooh yes just fixed!

**Review by annazhang2002 — APPROVED** [review]
looks great! seems good to merge

**Pull request merged**